frontend/components/common/Tags.js

```diff
@@ -11,8 +12,15 @@ import {
 
 export const Tag = s.span`
   margin: 0 4px 4px 0;
-  fontSize: .7em;
   font-weight: 600;
+  ${({ color }) =>
+    color &&
+    `
+    background-color: #${color} !important;
+    color: ${Color(`#${color}`)
+      .negate()
+      .hex()} !important;
```

Comment:
Yeah I think it would be good to use lighten and darken—can you maybe give a screenshot example of when the color is not neutral (in which case negating doesn't really do anything). As I can see, this will give us like orange text on blue background or something. If we use lighten and darken and do it enough ST it is an accessible contrast between the two colors, I think that will be more in line with how our other tags are styled.

Reply:
Changed!